### DISCUSSION TITLE
Thermalright Warframe 360 aio lcd

### DISCUSSION THREAD

**Discussion by windin101** [mathoudebine/turing-smart-screen-python#547] [General]
Hi 

This is a long shot but I though nothing ventured nothing gained

I have a Thermalright Warframe 360 aio, the pump lcd for which is a separate usb display attached to the pump block via magnets and has no functionality related to the pump at all. Its managed via the Thermawright TRCC software which lets you do some simple config with whats displayed. 

https://www.youtube.com/watch?v=X8oqqx4FrSA

Popping it open there is a single PCB connected to the display via a ribbon cable, the main IC is marked as XJE9XR000RDA which if I google it gives me nothing. The board is marked as 50.LMA01001A which is also a dead end.

USB info for it is 

![image](https://github.com/user-attachments/assets/99d19968-f24b-41e7-9d12-b5c4ad338339)

![image](https://github.com/user-attachments/assets/6f1366c4-117b-40c6-98ec-bbd93e83a6d7)

![image](https://github.com/user-attachments/assets/c5e359a8-04f6-4e9c-87f0-d660fe5af419)

![image](https://github.com/user-attachments/assets/72d67963-5bff-4288-8845-63600a853998)

**Comment by jezzaw007** [comment]
I too have a thermalright AIO
When i did a bit of research before i bought it, i thought this would likely work
I have Frozen Vision, I have got the display connected and it seems to be firing commands without any error, but I'm failing to get a display output, happy to work with you to try & figure this out. So its sending text or graphic to the display just not rendering, possibly a handshake issue, or incorrect raw format being sent
